app/components/shared/gcse_qualification_cards_component.html.erb

```diff
@@ -74,7 +74,13 @@
                   </dd>
                 <% end %>
               <% else %>
-                <% raise 'Neither `grade` nor `constituent_grades` found' %>
+                <div class="govuk-warning-text">
+                  <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+                  <strong class="govuk-warning-text__text">
+                    <span class="govuk-warning-text__assistive"></span>
+                    There is no grade recorded for this qualification.
+                  </strong>
+                </div>
```

Comment:
I don't think we need to use the warning text here. A simple message styled the same as the rest, and using a `<dd>` element, should be fine (and more accessible)?

```suggestion
                    <dd class="app-qualification__value">Not added</dd>
```